### PR TITLE
homework week1 DB

### PR DESCRIPTION
**Pull request by RimSebai**

### CODE REVIEW

File: myhw-week1/created-data.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+connection.query(sql, function (error, results) {
+  if (error) {
+    throw error;
+  }
+  console.log(results);
+});
+
+connection.query("use world")
+
+let countries_query = "create table if not exists countries(country_name varchar(20) unique, country_population bigint, continent varchar(20), surface_area int,primary key(country_name))"
```
Comment (unmeshvrije) [inline myhw-week1/created-data.js]: can country_name be a primary key ?

File: myhw-week1/db.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+let all_countries = "select * from countries order by surface_area desc"
+connection.query(all_countries, function (error, results) {
+  if (error) {
+    throw error;
+  }
+  console.log(results);
+});
+
+// Answer number (6)
+let netherlands_cities = "select countries.country_name, cities.city_name from cities inner join countries on cities.country = countries.country_name where countries.country_name = 'netherlands'"
```
Comment (unmeshvrije) [inline myhw-week1/db.js]: please explain the use of inner join.

File: myhw-week1/created-data.js (hunk 1 above)
Comment (RimSebai) [reply myhw-week1/created-data.js]: yes why not it is uniquely identifies  and contains UNIQUE values, right?

File: myhw-week1/db.js (hunk 2 above)
Comment (RimSebai) [reply myhw-week1/db.js]: I wanted to join two tables 
first i did specify which columns i want to display by using "select". 
second I used "from" to specify the tables i wanted to join. 
third I did specify which columns I wanted to much by using "on"  and the condition by using "where"

Comment (RimSebai) [reply myhw-week1/db.js]: ooh I had to use join to get only the matched data.

File: myhw-week1/created-data.js (hunk 1 above)
Comment (unmeshvrije) [reply myhw-week1/created-data.js]: I will accept it for this homework.

File: myhw-week1/db.js (hunk 2 above)
Comment (unmeshvrije) [reply myhw-week1/db.js]: Good explanation! I like that you know what you are doing exactly.

**Pull request closed without merging**